db/migrate/20150612135546_create_host_status.rb

```diff
@@ -0,0 +1,44 @@
+class CreateHostStatus < ActiveRecord::Migration
+  def self.up
+    create_table :host_status do |t|
+      t.string :type
+      t.integer :status, :default => 0, :null => false
+      t.references :host, :null => false
+      t.datetime :reported_at, :null => false
```

Comment:
is reported_at different from created_at for host_status? does it come from the report itself?